gui/slick/interfaces/default/inc_bottom.mako

```diff
@@ -66,11 +66,15 @@
         <!--
```

Comment:
I think it just used to be part of the submenu, before it was added to the navbar.